lib/validation.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Validation
+  SECRET_CODE_LENGTH = 4
+  MIN_LETTERS = 3
+  MAX_LETTERS = 20
+
+  def name_is_valid?(name)
```

Comment:
validation methods should be abstract:
`def valid_string_lenth?(input, min = MIN_LETTERS, max = MAX_LETTERS)`